snmp/datadog_checks/snmp/config.py

```diff
@@ -467,9 +474,9 @@ def get_table_symbols(mib, table):
             elif bulk_limit and len(symbols) > bulk_limit:
                 bulk_oids.append(table_oid)
             else:
-                all_oids.extend(symbols)
+                next_oids.extend(symbols)
```

Comment:
Why are these being separated out?  Maybe a comment to explain why they should be handled differently?

Reply:
Sure. The idea is that a get on those will never succeed, so we should skip that useless request.

Reply:
Went ahead and added a comment while merging master in here. @mgarabed Let me know if it's clear enough

Reply:
Also:

> Do a large number of metrics fall through the snmp_get check causing redundant queries?

From what I understand basically all metrics under the "symbols" section of a `table` metric entry would be in that case: we'd issue a first GET query on the column, not find anything, then issue a GETNEXT to get the correct OID.

AFAIU this PR only saves us one GET per table column — but I'm not sure if it would actually be _one GET per entry in the table and per table column_? @therve

Reply:
👍 - that comment is very helpful, thanks!